src/services/App.ts

```diff
@@ -33,13 +35,15 @@ export default class App {
     this.adminServer = deps(AdminServer)
     this.podManager = deps(PodManager)
     this.secret = deps(Secret)
+    this.ildcp = deps(Ildcp)
     this.money = deps(Money)
     this.backgroundValidatePeers = deps(BackgroundValidatePeers)
     this.selfTest = deps(SelfTest)
   }
 
   async start () {
     log.info('starting codiusd...')
+    await this.ildcp.init()
```

Comment:
how does this behave if the codius instance is in development mode? Because the moneyd.start call is in an `if (!this.config.devMode)` block it kinda seems like it expects there to sometimes not be moneyd access.

Reply:
It uses default/dummy values from `Config` (doesn't use moneyd)
https://github.com/codius/codiusd/pull/144/files#diff-ce8c933b2833c6b6aee9bca716ee79c6R20
https://github.com/codius/codiusd/pull/144/files#diff-050bb2cfa5eaf49e7b479d21ad5be0fbR79
Otherwise, endpoints that use `Ildcp` wouldn't work with `devMode`.
Alternatively, I could move the `devMode` ILDCP assignment from `init()` to the `Ildcp` constructor.